cmd/warpforge/check.go

```diff
@@ -78,34 +89,43 @@ func cmdCheck(c *cli.Context) error {
 		return fmt.Errorf("no input files provided")
 	}
 	ctx := c.Context
-
+	pwd, err := os.Getwd()
+	if err != nil {
+		return err
```

Comment:
Should this return a wapi error? Something like `return serum.Errorf(wfapi.ECodeUnknown, "unable to get pwd: %w", err)`

Reply:
Yeah. Serum doesn't detect these so they are pretty inconsistent across everything in `cmd/*`. This should improve as we move things under the plumbing package since we'll have more consistent command setup/execution and the business logic entry point will be an exported function that serum checks.

Reply:
Adding an `Errors:` comment to our command entry points would coerce serum into checking these functions which would probably be a good idea for a separate PR.